lib/auth/apiserver.go

```diff
@@ -508,6 +508,8 @@ type WebSessionReq struct {
 	// Switchback is a flag to indicate if user is wanting to switchback from an assumed role
 	// back to their default role.
 	Switchback bool `json:"switchback"`
+	// UpdateUserTraits is a flag to indicate if user traits need to be updated.
+	UpdateUserTraits bool `json:"update_user_traits"`
```

Comment:
Nit: Would it make sense to call this a more generic `ReloadUser`? This way we can update more than just traits in future if need be.